utils/rpms/daos.spec

```diff
@@ -637,6 +639,13 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Mon May 12 2025  Tomasz Gromadzki <tomasz.gromadzki@hpe.com> 2.7.101-9
```

Comment:
This also requires changing https://github.com/daos-stack/daos/blob/6d3a364fa8cb820cdcbd6bfd151ae6aceebea164/utils/rpms/daos.spec#L26 to
```
Release:       9%{?relval}%{?dist}
```

Reply:
Fixed.